YAFCmodel/Model/ProductionTableContent.cs

```diff
@@ -181,6 +181,15 @@ public class RecipeRow : ModelObject<ProductionTable>, IModuleFiller, IGroupedEl
         public bool hierarchyEnabled { get; internal set; }
         public int tag { get; set; }
 
+        public RowHighlighting highlighting =>
```

Comment:
Our editorconfig states the following
```
# Expression-bodied members
csharp_style_expression_bodied_methods = false:suggestion
csharp_style_expression_bodied_constructors = false:suggestion
csharp_style_expression_bodied_operators = false:suggestion
csharp_style_expression_bodied_properties = true:suggestion
csharp_style_expression_bodied_indexers = true:suggestion
csharp_style_expression_bodied_accessors = true:suggestion
```
I think `highlighting` falls under `csharp_style_expression_bodied_properties`, and we currently say to use lambdas for it. 

In the row that you pointed out, making it non-lambda would quicken my understanding of that piece of code. 

I'll check if setting all of them to `false:suggestion` makes sense.

Reply:
I'm not sure if a non-lambda would be nicer:

```csharp
public RowHighlighting highlighting {
    get {
        return tag switch {
            1 => RowHighlighting.Green,
            2 => RowHighlighting.Yellow,
            3 => RowHighlighting.Red,
            4 => RowHighlighting.Blue,
            _ => RowHighlighting.None
        };
    }
}
```

BTW, I explicitly used the switch expression, since a switch statement would mean more lines without any benefit:

```csharp
public RowHighlighting highlighting {
    get {
        switch (tag) {
            case 1:
                return RowHighlighting.Green;
            case 2:
                return RowHighlighting.Yellow;
            case 3:
                return RowHighlighting.Red;
            case 4:
                return RowHighlighting.Blue;
            default:
                return RowHighlighting.None;
        }
    }
}
```
I'm a big fan of the new switch expression, but I understand that it might seem unfamiliar if you're not accustomed to it.
The question is: Are you deterred by the switch expression itself or by the expression body? ;-)

Reply:
for me the switch expression _or_ lambda is fine. Having both got me confused :smile: (I am not a professional C# dev, so I am still unfamiliar with all its details)

Reply:
Lambdas inside a switch are fine with me.

Reply:
Ok, then I suppose we keep it as it is in the PR.